src/Surfnet/StepupMiddleware/ApiBundle/Doctrine/Type/SelfVetOptionType.php

```diff
@@ -20,14 +20,15 @@
 
 use Doctrine\DBAL\Platforms\AbstractPlatform;
 use Doctrine\DBAL\Types\ConversionException;
+use Doctrine\DBAL\Types\IntegerType;
 use Doctrine\DBAL\Types\Type;
 use Surfnet\Stepup\Configuration\Value\SelfVetOption;
 use TypeError;
 
 /**
  * Custom Type for the SelfVetOption Value Object
  */
-class SelfVetOptionType extends Type
+class SelfVetOptionType extends IntegerType
```

Comment:
Should these options not also have BooleanType given they are boolean values?

Reply:
The db value is an int so this seemed correct and also didn't result in a migration when running a diff.